common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -802,3 +785,44 @@ describe 'Problem', ->
 
       # verify that codemirror textarea has correct `aria-describedby` attribute value
       expect($(CodeMirrorTextArea).attr('aria-describedby')).toEqual('cm-editor-exit-message-101 status_101')
+
+
+  describe 'show answer button', ->
+
+    radioButtonProblemHtml = readFixtures('radiobutton_problem.html')
```

Comment:
@Rabia23 why do we need to check radio button and checkbox seperately? Both seem to be using the same logic.